penny_u_django/home/templates/home/index.html

```diff
@@ -15,6 +24,11 @@
             <div class="overlay-text">
                 <h1 class="display-2">Penny University</h1>
                 <h3 class="mt-4">A peer to peer learning community</h3>
+                <div class="text-center mt-5">
+                    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#formModal">
```

Comment:
Let's make this a large button, so it stands out more. https://getbootstrap.com/docs/4.3/components/buttons/#sizes

Reply:
I would make this button large & add btn-block class also.